src/shipit_static_analysis/tests/conftest.py

```diff
@@ -34,33 +44,37 @@ def mock_config():
     )
 
     from shipit_static_analysis.config import settings
-    settings.setup('test')
+    tempdir = tempfile.mkdtemp()
```

Comment:
You might want to yield `settings`, so that you can remove the temporary directory.